libarchive/entry.py

```diff
@@ -61,17 +61,36 @@ def isfifo(self):
 
     @property
     def islnk(self):
+        return bool(ffi.entry_hardlink_w(self._entry_p))
+
+    @property
+    def issym(self):
         return self.filetype & 0o170000 == 0o120000
```

Comment:
Typically null, but it is undefined. At the time you read some entry the linked-to-file is by construction something not yet known until you read the corresponding file. See https://github.com/libarchive/libarchive/wiki/Hardlinks for details